meta/FSMathLink.m

```diff
@@ -245,6 +249,9 @@
 CreateSpectrumDecaysInterface[modelName_] :=
     "virtual void " <> CreateSpectrumDecaysCalculationName[] <> "(const softsusy::QedQcd&, const Physical_input&, const FlexibleDecay_settings&) = 0;";
 
+CreateSpectrumUnitarityInterface[modelName_] :=
+    "virtual std::tuple<bool, double, double, Eigen::MatrixXcd> calculate_unitarity() = 0;";
```

Comment:
I'd recommend to use a `struct` instead of a `tuple`, because a struct has "named" elements, which improves readability.

Reply:
I agree. I was also thinking about it. This tuple got out of hand (it started with 2 elements). I'll make the change.

Reply:
Fixed in a756dd999fad3f0bf06a9b5460987428e744e6cd.